docs/samples/browser-plugin-meetings/app.js

```diff
@@ -456,7 +456,6 @@ function joinMeeting({ withMedia, withDevice } = { withMedia: false, withDevice:
 
   const joinOptions = {
     pin: meetingsJoinPinElm.value,
-    captcha: meetingsJoinCaptchaElm.value,
```

Comment:
I don't see captcha being read from `joinOptions` when follow the` meeting.join() `implementation, So removed, Thanks for pointing this marcin